### PR TITLE
Update no_unions flag for nanopb gen and update ONNX proto files

### PR DESCRIPTION
**Pull request by jamesr66a**

### CODE REVIEW

File: torch/csrc/jit/export.cpp [hunk 1]
```diff
@@ -128,8 +128,8 @@ void addAttribute(onnx::NodeProto * n_p, jit::Node * n, jit::Symbol name) {
   }
 }
 
-void encodeTypeProtoTensorType(onnx::TypeProtoTensorTypeProto* tensor_type, Value* n) {
-  onnx::TypeProtoTensorShapeProto* shape = tensor_type->mutable_shape();
+void encodeTypeProtoTensorType(onnx::TypeProtoTensor* tensor_type, Value* n) {
+  onnx::TensorShapeProto* shape = tensor_type->mutable_shape();
```
Comment (ezyang) [inline torch/csrc/jit/export.cpp]: Yissss

**Review by ezyang — APPROVED** [review]
What's up with onnx-fb-universe? I thought this shouldn't be BC breaking.

**Comment by jamesr66a** [comment]
@ezyang  it looks like the IR version change here: https://github.com/onnx/onnx/pull/186/files#diff-90937e0f36f31f454e47fc24e919e006R79

**Comment by ezyang** [comment]
OK, I don't see a companion PR for onnx-fb-universe. Please ping when you have it.

**Pull request merged**